test/ftp_object.jl

```diff
@@ -35,10 +33,10 @@ function expected_output(active::Bool)
     close(ftp)
 end
 
-function copy_and_wait(f::Function, files...; timeout=30)
+function copy_and_wait(func::Function, files...; timeout=30)
```

Comment:
?

Reply:
Disambiguates between this `f` and the `f` in the loop below

Reply:
yeah I dunno just felt weird to redefine f

Reply:
Ah, I didn't see that it was being used again, don't mind me.